content/product-engineering/planning.md

```diff
@@ -0,0 +1,147 @@
+# Product and engineering planning
+
```

Comment:
Proposing to add a section about our philosophy/principles. 

##Planning philosophy and principles 
Our planning process defines how we meet and execute to create our annual and quarterly goals. Our philosophy is based on the following principles:

- Clear direction: by setting clear direction on the department level we are able to stay focused throughout the year and have appropriate trade-off discussions if necessary. 

- Coupling insights: our teams are the domain experts and their valuable insights coupled with the overall company strategy are critical to both enable and drive our strategic planning.

- Creating impact : to set ourselves up for success we want to understand and be able to measure the impact and value the work we commit to will have on customers, our internal teams, and the overall success of Sourcegraph.

Reply:
@msennett22 FYI there is a philosophy section at `/product-engineering/planning#philosophy` (could be brand new, not sure) and it touches on some of these.